src/components/InfoViewer/InfoViewer.scss

```diff
@@ -54,6 +54,8 @@
     &__value {
         display: flex;
 
+        min-width: 120px;
```

Comment:
120px, using the current font settings, makes Date standard string representation display in two rows, the first being date and the second being time. No other reason.